webpack/assets/javascripts/react_app/components/Pagination/__snapshots__/Pagination.test.js.snap

```diff
@@ -3,6 +3,8 @@
 exports[`Pagination rendering from erb renders layout 1`] = `
 <Paginator
   className="col-md-12"
+  disableNext={false}
```

Comment:
I am not sure, but wasn't this snapshot already changed in one of the last upgrades? :stuck_out_tongue_closed_eyes:

Reply:
also for 1.22?